edsnlp/pipelines/qualifiers/history/history.py

```diff
@@ -155,6 +196,9 @@ def set_extensions(cls) -> None:
         if not Span.has_extension("history_cues"):
             Span.set_extension("history_cues", default=[])
 
+        if not Span.has_extension("recent_cues"):
+            Span.set_extension("recent_cues", default=[])
+
```

Comment:
I'm not sure about the `recent_cues` extension, couldn't we just ignore a date completely if it doesn't pass the timedelta requirement? Or if we need the date to explain why a atcd marker was not enough to qualify an entity as history, could we instead group all dates under a single `date_cues` ? What do you think ?

Reply:
Just added an explanatory comment:

```python
# Store history mentions responsible for the history entity's character
if not Span.has_extension("history_cues"):
    Span.set_extension("history_cues", default=[])

# Store recent mentions responsible for the non-antecedent entity's character
if not Span.has_extension("recent_cues"):
    Span.set_extension("recent_cues", default=[])
```